cli/args/flags.rs

```diff
@@ -213,6 +213,14 @@ pub struct RunFlags {
 }
 
 impl RunFlags {
+  #[cfg(test)]
+  pub fn new_default(script: String) -> Self {
```

Comment:
I kept this change because I think it will make things easier in the future if we ever add another flag to this sub command.